services/build.gradle

```diff
@@ -6,14 +6,15 @@ sourceSets {
                          '../services-optimization/src/main/java', '../services-staticmap/src/main/java',
                          '../services-speech/src/main/java', '../services-tilequery/src/main/java',
                          '../services-route-tiles/src/main/java', '../services-directions-refresh/src/main/java',
-                         '../services-isochrone/src/main/java']
+                         '../services-isochrone/src/main/java', '../services-matching/src/main/java']
```

Comment:
include `mapmatching` to `services` api-ref docs  

cc @Guardiola31337

Reply:
@RingerJK @Guardiola31337  When will we see the new map matching docs on https://docs.mapbox.com/android/java/api-reference/?

Reply:
They're already available under [mapbox-java-services](https://docs.mapbox.com/android/java/api/libjava-services/6.4.0/) e.g. [`MapMatchingService`](https://docs.mapbox.com/android/java/api/libjava-services/6.4.0/com/mapbox/api/matching/v5/MapMatchingService.html) 🚀